botocore/operation.py

```diff
@@ -86,8 +113,25 @@ def call(self, endpoint, **kwargs):
         self.session.emit(event, endpoint=endpoint,
                           model=self.model,
                           params=request_dict,
-                          operation=self)
+                          operation=self,
+                          request_signer=signer)
```

Comment:
Why is the request_signer added to before-call kwargs?  Seems unnecessary to me.